components/lsp/MintForm.tsx

```diff
@@ -180,6 +188,14 @@ const MintForm: FC<Props> = ({
     userNeedsToApprove,
   ]);
 
+  if (web3Provider && !isUserConnectedToContractChain) {
+    return (
+      <SwitchWalletContainer>
+        <SwitchWalletLsp targetChainId={chainId} />
+      </SwitchWalletContainer>
```

Comment:
can u add a screenshot of what this looks like?

Reply:
Sure
<img width="1115" alt="Screenshot 2022-02-07 at 15 20 10" src="https://user-images.githubusercontent.com/89395931/152795736-110f17e3-4535-490b-a94c-e0e54abdaffd.png">